core/src/verification/queue/kind.rs

```diff
@@ -181,19 +191,15 @@ pub mod blocks {
             use crate::views::BlockView;
 
             let header = BlockView::new(&bytes).header();
+            let mem_usage = rlp::encode(&header).to_vec().len() + bytes.len();
```

Comment:
Isn't `rlp::encode(&header).to_vec().len()` `header.mem_usage() `?

Reply:
It is defined in `mod headers`, not `mod blocks`.